app-template/config-template.xml

```diff
@@ -51,14 +47,17 @@
     <plugin name="cordova-plugin-android-fingerprint-auth" spec="1.4.3" />
     <plugin name="cordova-plugin-device" spec="2.0.1" />
     <plugin name="cordova-plugin-globalization" spec="1.0.9" />
-    <plugin name="cordova-plugin-qrscanner" spec="2.5.0" />
+    <plugin name="cordova-plugin-qrscanner" spec="2.6.0" />
+    <plugin name="cordova-plugin-add-swift-support" spec="1.7.2" />
```

Comment:
why we need this?

Reply:
`cordova-plugin-qrscanner` requires `cordova-plugin-add-swift-support`, but it 's not fixed version. Check this line:
```

Reply:
Install a fixed version of `cordova-plugin-add-swift-support` in our side, would avoid to install any update of this.

Reply:
ack